evalml/tests/data_checks_tests/test_highly_null_rows_data_check.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+import pandas as pd
+import pytest
+import woodwork as ww
+
+from evalml.data_checks import (
+    DataCheckAction,
+    DataCheckActionCode,
+    DataCheckMessageCode,
+    DataCheckWarning,
+    HighlyNullRowsDataCheck
+)
+
+
+def test_highly_null_data_check_init():
+    highly_null_check = HighlyNullRowsDataCheck()
+    assert highly_null_check.pct_null_threshold == 0.5
+
+    highly_null_check = HighlyNullRowsDataCheck(pct_null_threshold=0.0)
+    assert highly_null_check.pct_null_threshold == 0
+
+    highly_null_check = HighlyNullRowsDataCheck(pct_null_threshold=0.5)
+    assert highly_null_check.pct_null_threshold == 0.5
+
+    highly_null_check = HighlyNullRowsDataCheck(pct_null_threshold=1.0)
+    assert highly_null_check.pct_null_threshold == 1.0
+
+    with pytest.raises(ValueError, match="pct_null_threshold must be a float between 0 and 1, inclusive."):
+        HighlyNullRowsDataCheck(pct_null_threshold=-0.1)
+    with pytest.raises(ValueError, match="pct_null_threshold must be a float between 0 and 1, inclusive."):
+        HighlyNullRowsDataCheck(pct_null_threshold=1.1)
+
```

Comment:
I really like these tests.  Super clear to read and infer the behavior of the datacheck.  Great job.